### DISCUSSION TITLE
Is there a documentation or Reference for the Whisper API ??

### DISCUSSION THREAD

**Discussion by DantrazTrev** [openai/whisper#525] [Q&A] [answered]
Hei ,
So I am unable to find like a doc or ref sheet for the whisper's API . Is there any such thing that I might be missing out??

**Comment by jongwook** [comment, marked as answer]
There's none at the moment, other than the docstrings:

https://github.com/openai/whisper/blob/9f70a352f9f8630ab3aa0d06af5cb9532bd8c21d/whisper/transcribe.py#L31-L72

https://github.com/openai/whisper/blob/9f70a352f9f8630ab3aa0d06af5cb9532bd8c21d/whisper/decoding.py#L677-L696
